scons/site_tools/hosted.py

```diff
@@ -42,6 +42,8 @@ def generate(env, **kw):
 		env['LIBS'] = ['boost_thread', 'boost_system', 'pthread']
 	elif env['HOSTED_DEVICE'] == 'darwin':
 		env['LIBS'] = ['boost_thread-mt', 'boost_system']
+	elif env.System() == 'windows':
+		env['LIBS'] = ['winpthread', 'boost_thread-mt', 'boost_system', 'wsock32', 'mingw32', 'SDLmain', 'SDL', 'winmm', 'gdi32', 'dxguid', 'opengl32', 'gdi32', 'winmm', 'gdi32']
```

Comment:
You are right. Artefact while I was testing if this was changing the SDL linkage (dx vs GL)